Evaluation_metrics/em.py

```diff
@@ -0,0 +1,67 @@
+"""Evaluation metrics that will be used to assess the output of the trained model compared
+to references codes."""
+
+from typing import Tuple, List, Any
+from Levenshtein import distance
+from datasets import load_metric
+from nltk.translate.bleu_score import sentence_bleu
+
+
+class EM:
+    """
+    Evaluation metrics that will be used to assess the output of the trained model compared
+    to reference codes.
+    """
+
+    def __init__(self) -> None:
+        """
+        Initializes an instance of the EM class.
+        """
+        pass
+
+    def edit_distance(self, generated_code: str, reference_code: str) -> int:
+        """
+        Computes the Levenshtein edit distance between the generated code and the reference code.
+
+        Parameters:

```

Comment:
Use the following tool to generate the docstrings for you 
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring